### PR TITLE
Add custom code for Reddit archiving

### PR DESCRIPTION
**Pull request by CorentinB**

### CODE REVIEW

File: internal/pkg/crawl/assets.go [hunk 1]
```diff
@@ -30,7 +31,7 @@ func (c *Crawl) extractAssets(base *url.URL, item *queue.Item, doc *goquery.Docu
 	doc.Find("[data-item]").Each(func(index int, item *goquery.Selection) {
 		dataItem, exists := item.Attr("data-item")
 		if exists {
-			URLsFromJSON, _ := getURLsFromJSON(dataItem)
+			URLsFromJSON, _ := extractor.GetURLsFromJSON(dataItem)
```
Comment (equals215) [inline internal/pkg/crawl/assets.go:34]: no err check?

File: internal/pkg/crawl/assets.go [hunk 2]
```diff
@@ -136,7 +137,7 @@ func (c *Crawl) extractAssets(base *url.URL, item *queue.Item, doc *goquery.Docu
 			scriptType, exists := item.Attr("type")
 			if exists {
 				if scriptType == "application/json" {
-					URLsFromJSON, _ := getURLsFromJSON(item.Text())
+					URLsFromJSON, _ := extractor.GetURLsFromJSON(item.Text())
```
Comment (equals215) [inline internal/pkg/crawl/assets.go:140]: no err check?

File: internal/pkg/crawl/assets.go [hunk 3]
```diff
@@ -184,7 +185,7 @@ func (c *Crawl) extractAssets(base *url.URL, item *queue.Item, doc *goquery.Docu
 					}
 
 					if len(jsonContent[1]) > payloadEndPosition {
-						URLsFromJSON, _ := getURLsFromJSON(jsonContent[1][:payloadEndPosition+1])
+						URLsFromJSON, _ := extractor.GetURLsFromJSON(jsonContent[1][:payloadEndPosition+1])
```
Comment (equals215) [inline internal/pkg/crawl/assets.go:188]: no err check?

**Review by equals215 — CHANGES_REQUESTED** [review]
LGTM after changes if necessary

**Comment by CorentinB** [comment]
About err checks, I'm not sure it's useful, not sure we want to log them and we have no action to take on them. What do you think?

**Pull request merged**